backend/inspirehep/alembic/3fd6471bb960_remove_legacy_records_mirror_table.py

```diff
@@ -0,0 +1,51 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2016-2018 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""remove_legacy_records_mirror_table"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "3fd6471bb960"
+down_revision = "72d010d89702"
+branch_labels = ()
+depends_on = None
+
+
+def upgrade():
```

Comment:
We should also update https://github.com/inspirehep/inspirehep/blob/f28e872812dcde800f6b8090691a687c79896b9a/backend/tests/integration/alembic/test_alembic.py#L15 :)